app/src/main/java/de/danoeh/antennapod/fragment/preferences/ImportExportPreferencesFragment.java

```diff
@@ -181,70 +181,67 @@ private void exportDatabase() {
+                                .setType("*/*");
                         startActivityForResult(intent, REQUEST_CODE_RESTORE_DATABASE);
                     } else {
-                        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
-                        intent.setType("*/*");
+                        Intent intent = new Intent(Intent.ACTION_GET_CONTENT)
+                                .setType("*/*");
                         startActivityForResult(Intent.createChooser(intent,
                                 getString(R.string.import_select_file)), REQUEST_CODE_RESTORE_DATABASE);
                     }
-                }
-        );
-
-        // create and show the alert dialog
-        builder.show();
+                })
+                .show();
     }
 
     private void showDatabaseImportSuccessDialog() {
-        AlertDialog.Builder d = new AlertDialog.Builder(getContext());
-        d.setMessage(R.string.import_ok);
-        d.setCancelable(false);
-        d.setPositiveButton(android.R.string.ok, (dialogInterface, i) -> PodcastApp.forceRestart());
-        d.show();
+        new AlertDialog.Builder(getContext())
+                .setMessage(R.string.import_ok)
+                .setCancelable(false)
+                .setPositiveButton(android.R.string.ok, (dialogInterface, i) -> PodcastApp.forceRestart())
+                .show();
     }
 
     private void showExportSuccessDialog(final String path, final Uri streamUri) {
-        final AlertDialog.Builder alert = new AlertDialog.Builder(getContext());
-        alert.setNeutralButton(android.R.string.ok, (dialog, which) -> dialog.dismiss());
-        alert.setTitle(R.string.export_success_title);
-        alert.setMessage(getContext().getString(R.string.export_success_sum, path));
-        alert.setPositiveButton(R.string.send_label, (dialog, which) -> {
-            Intent sendIntent = new Intent(Intent.ACTION_SEND);
-            sendIntent.putExtra(Intent.EXTRA_SUBJECT, getString(R.string.opml_export_label));
-            sendIntent.putExtra(Intent.EXTRA_STREAM, streamUri);
-            sendIntent.setType("text/plain");
-            sendIntent.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
-            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
-                List<ResolveInfo> resInfoList = getContext().getPackageManager()
-                        .queryIntentActivities(sendIntent, PackageManager.MATCH_DEFAULT_ONLY);
-                for (ResolveInfo resolveInfo : resInfoList) {
-                    String packageName = resolveInfo.activityInfo.packageName;
-                    getContext().grantUriPermission(packageName, streamUri, Intent.FLAG_GRANT_READ_URI_PERMISSION);
-                }
-            }
-            getContext().startActivity(Intent.createChooser(sendIntent, getString(R.string.send_label)));
-        });
-        alert.create().show();
+        new AlertDialog.Builder(getContext())
```

Comment:
In this case, I find it easier to read the original version. It needs less indentation and shows more clearly what belongs together.

Reply:
reverted